### ISSUE TITLE
When app is running, Command + Number doesn't work globally

### ISSUE THREAD

**Issue by squalvj** [toeverything/AFFiNE#7826]
### What happened?

When I run the app, regardless it is minimized or ran in the background.
It prevents me from pressing `Command + Number` simultaneously globally, if I close the app then I can press `Command + Number` again.

This is annoying as I used to navigate between tabs in my browser or other software by pressing `Command + Number`, but I couldn't do it unless I close the Affine desktop.

### Distribution version

macOS x64 (Intel)

### What browsers are you seeing the problem on if you're using web version?

Chrome

### Are you self-hosting?

- [ ] Yes

### Relevant log output

```shell
Affine desktop won't prevent from pressing `Command + Number` globally if it's minimized.
```

### Anything else?

AFFiNE Desktop (Mac)
Version: 0.16.1

Editor Version
0.17.0-canary-202408080742-0cbcb73

**Comment by pengx17** [comment]
Thanks for reporting!

**Comment by squalvj** [comment]
Godspeed !
Appreciate it man!

**Comment by forehalo** [comment]
will be fixed in 0.16.2
